aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -98,6 +101,80 @@ Tensor _dim_arange(const Tensor& like, int64_t dim) {
   return at::arange(like.size(dim), like.options().dtype(at::kLong));
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~ complex / complex_polar ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TensorIterator complex_op(
+    Tensor& out,
+    const Tensor& a,
+    const Tensor& b,
+    bool check_mem_overlap) {
+  return TensorIteratorConfig()
+      .set_check_mem_overlap(check_mem_overlap)
+      .add_output(out)
+      .add_input(a)
+      .add_input(b)
+      .allow_cpu_scalars(true)
```

Comment:
Does this really support calls like `torch.complex(t, 5)`?

Reply:
hmm oh it doesn't actually.

Reply:
That's OK. You can just remove this line, I think.